docs/content/documentation/getting-started/cli-usage.md

```diff
@@ -1,24 +1,24 @@
 +++
 title = "CLI usage"
-weight = 2
+weight = 20
```

Comment:
Is that intended?

Reply:
It was not intended for the "next" branch. I'm reorganizing the documentation in a "docs-next' branch on my fork. I changed the weights to multiples of 10 in case I need to add new sections (for example, I added an "Overview" section to "Getting Started"). Please revert.